compiler/lib/source_map.ml

```diff
@@ -301,7 +301,7 @@ let merge = function
 
 (* IO *)
 
-let json t =
+let json ?replace_mappings t =
```

Comment:
Can you explain the usecase for such api ?

Reply:
In wasm_of_ocaml, when linking, it was observed that parsing the `mappings` field of sourcemaps takes a big toll on the overall performance, and was unnecessary since the mappings are not modified. So this is a workaround to sidestep this parsing, the cleaner way being in #1617.